### PR TITLE
storage dumper

### PR DESCRIPTION
**Pull request by Ashuaidehao**
Close https://github.com/neo-project/neo-modules/issues/736.
Alternative https://github.com/neo-project/neo-modules/pull/789.

### CODE REVIEW

**Review by superboyiii — APPROVED** [review]
Works as expected and fixed that data missing when exit.

**Review by vncoelho — APPROVED** [review]

File: src/StorageDumper/config.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "PluginConfiguration": {
+    "BlockCacheSize": 1000,
+    "HeightToBegin": 0,
+    "Exclude": [ -4 ]
```
Comment (gsmachado) [inline src/StorageDumper/config.json:5]: what's "Exclude"?

where's this documented?

Comment (Ashuaidehao) [reply src/StorageDumper/config.json:5]: Exclude Contract's ID, same use as the old StatesDumper plugin.
Ignore LedgerContract(-4) storage changes by default.

**Comment by vncoelho** [comment]
@Ashuaidehao, we missed what to do with StatesDumper and the unification of both.

**Comment by vncoelho** [comment]
@Ashuaidehao, can you address the unification of both?

**Pull request merged**